app/components/post/index.js

```diff
@@ -18,11 +18,38 @@ function mapStateToProps(state, ownProps) {
     const {config, license} = state.entities.general;
     const roles = getCurrentUserId(state) ? getCurrentUserRoles(state) : '';
 
+    let isFirstReply = true;
```

Comment:
can we make this into a function? also do we need to run this every time the state changes?

Reply:
Yeah, I'll take a look

Reply:
Following up here, I set up a benchmark, and the function is running fast enough that we're not going to make further changes. Any changes I did try also seemed to either have no changes or make it worse